packages/openapi3/README.md

```diff
@@ -49,6 +49,35 @@ union MyUnion {
 
 ```
 
+## Decorators
+
+- [@useRef](#useref)
+- [@oneOf](#oneof)
+
+### @useRef
+
+Syntax:
+
+```
+@useRef(urlString)
+```
+
+`@useRef`
+
+`@useRef` can only be applied to model or model property types.
```

Comment:
"It is used to replace the Cadl model type in emitter output with a pre-existing named OpenAPI schema."